### ISSUE TITLE
`lines-around-directive` rule should replaced with `padding-line-between-statements`

### ISSUE THREAD

**Issue by mvorisek** [airbnb/javascript#2689]
see https://github.com/eslint/eslint/issues/16629#issuecomment-1341641006 from the eslint maintainer

**Comment by HemanthPaila** [comment]
Hi there,

I want to contribute to this project and take on this issue as my first task. Could you please assign it to me? I am excited to work towards resolving it and will keep the team updated on my progress.

Thanks!

**Comment by ljharb** [comment]
@HemanthPaila theres no need to request assignment; most open source projects don’t operate that way. Just make a PR and link it to the issue.

**Comment by HemanthPaila** [comment]
Thanks @ljharb . I am on it.

**Comment by HemanthPaila** [comment]
I made a PR on this issue. could you please review and merge it?

**Comment by HemanthPaila** [comment]
Hello sir i made another PR could you please review it once?

**Comment by ljharb** [comment]
@HemanthPaila please don't make duplicate PRs - now both #2756 and #2757 must remain open and be kept in sync. I've reviewed it, and in the future you don't need to ask for review, you can just wait more than a few minutes :-)

**Comment by HemanthPaila** [comment]
Sir could you please clarify my doubt:
      - What I want to add to the issue for a successful merge.
      - I regret removing the entry, but I will correct the issue by returning it to the appropriate place.

**Comment by speedywarp** [comment]
is the issue resolved?

**Comment by ljharb** [comment]
No, the linked PR, https://github.com/airbnb/javascript/pull/2757, has unresolved comments.
